loadtest/store/memstore/store.go

```diff
@@ -190,6 +190,14 @@ func (s *MemStore) RemoveChannelMember(channelId string, userId string) error {
 	return nil
 }
 
+func (s *MemStore) RemoveTeamMember(teamId string, userId string) error {
+	if s.teamMembers[teamId] == nil {
+		return nil
+	}
```

Comment:
I think we are safe without this block. You can call delete on an empty/nil map.